src/browsers/schema/content/about.md

```diff
@@ -0,0 +1,46 @@
+
+# Collections
+
+## Ashkenazi Jewish schizophrenia study
+David C. Glahn, Ann E. Pulver
+
+## Danish iPSYCH initiative
+Anders D. Børglum, Jonas Bybjerg-Grauholm, Jakob Grove, David M. Hougaard, Francesco Lescai, Preben Bo Mortensen, Merete Nordentoft, Thomas M. Werge
+
+## McLean psychosis study
+Bruce M. Cohen, Mei-Hua Hall, Steven A. McCarroll, Dost Ongur
+
+## Pritzker Neuropsychiatric Disorders Research Consortium (Pritzker NDRC)
+Huda Akil, Jack D. Barchas, William E. Bunney, William F. Byerley, Lynn DeLisi, Francis S. Lee, Richard M. Myers, Brandi Rollins, Alan Schatzberg, Marquis P. Vawter, Stanley J. Watson
+
+## Swedish schizophrenia study
+Christina M. Hultman, Jordan W. Smoller, Patrick F. Sullivan
+
+## Taiwanese trios study
```

Comment:
Are these intended to be displayed as separate lines? As is, everything in this section will be displayed as one long line.

Reply:
They are not. I have added some spaces to fix it hopefully..